CHANGELOG

```diff
@@ -8,18 +8,638 @@ INIFileEditor - graphical parameter editor for INI files
 Parameters - list of algorithm or TOPP tool parameters that changed in this release
 
 ------------------------------------------------------------------------------------------
-----                                OpenMS 1.12.0                                     ----
+----                                OpenMS 2.0                                        ----
 ------------------------------------------------------------------------------------------
-Release date: unknown
+Release date: January 2015
 
-OpenMS 1.12.0 is the first release after the switch to git and a complete overhaul of the
-build system. Furthermore we removed the dependency to GSL in OpenMS 1.12.0 and replaced
-the functionality with Eigen3 and Wildmagic.
+OpenMS 2.0 is the first release after the switch to git and a complete overhaul of the
+build system. It introduces a considerable number of new features and bug fixes.
+Furthermore, we removed the dependency to GSL and replaced the functionality using
+Eigen3 and Wildmagic.
+
+File formats:
+  - mzQuantML support (experimental)
+  - mzIdentML support (experimental)
+  - mzTab support (experimental)
+
+TOPPView:
+  - Support for visualizing mass fingerprinting hits from featureXML along with their raw spectra in MS1
+  - Improved "Tools" -> "Goto" dialog
+  - Improved display of m/z, RT, and intensity values 1D and 2D view
+
+New tools:
+  - FeatureFinderIdentification (TOPP)
+  - FeatureFinderMultiplex (TOPP)
+  - FidoAdapter (TOPP)
+  - LowMemPeakPickerHiRes (UTIL)
+  - LowMemPeakPickerHiRes_RandomAccess (UTIL)
+  - MRMTransitionGroupPicker (UTIL)
+  - MSGFPlusAdapter (TOPP)
+  - MetaboliteSpectralMatcher (UTIL)
+  - OpenSwathWorkflow (UTIL)
+  - PeakPickerIterative (UTIL)
+  - RTAnnotator (UTIL)
+  - SimpleSearchEngine (UTIL)
+  - TopPerc (TOPP)
+
+Deprecated tools:
+  - DBExporter (TOPP)
+  - DBImporter (TOPP)
+  - FeatureFinderRaw (TOPP)
+  - SILACAnalyzer (TOPP)
+
+General:
+  - Removed GSL dependencies
+
+Third party software:
+  - Added Fido support
+  - Added MS-GF+ support
+
+Changes to the Build System / Package System:
+  - Restructured repository layout
+  - Added support for Travis CI
+  - Simplified pyOpenMS build system
+
+Changed parameters:
```

Comment:
can we have this in a separate file? it will be unreadable very fast otherwise ...